### PR TITLE
MGMT-16880: assisted-service config for ocm 2.10

### PR DESCRIPTION
**Pull request by adriengentil**

### CODE REVIEW

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.10.yaml [hunk 1]
```diff
@@ -0,0 +1,725 @@
...
+- as: mirror-vcsref-image
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: assisted-service
+    env:
+      RELEASE_TAG_PREFIX: ocm-2.10
+    test:
+    - ref: assisted-baremetal-images-publish
+- as: push-pr-image
```
[the remark this answers is not in the document]
Comment (osherdp) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.10.yaml:176]: you know what? let's remove it from everywhere, it's not worth the future risk

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 2]
```diff
@@ -399,7 +399,8 @@ tests:
     env:
       PACKET_PLAN: c3.medium.x86
     workflow: assisted-baremetal-operator-ztp
-- as: e2e-ai-operator-ztp-3masters
+- always_run: false
```
[the remark this answers is not in the document]
Comment (adriengentil) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:402]: oh right, I did it mechanically

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.10.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (adriengentil) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.10.yaml]: done!

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (adriengentil) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml]: done!

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.8.yaml [hunk 3]
```diff
@@ -161,16 +161,7 @@ tests:
...
-  optional: true
-  steps:
-    dependencies:
-      SOURCE_IMAGE_REF: assisted-service
-    env:
-      RELEASE_TAG_PREFIX: ocm-2.8
-    test:
-    - ref: assisted-baremetal-images-publish
-- as: operator-publish
+- as: oserator-publish
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.8.yaml:164]: typo

**Review by osherdp — CHANGES_REQUESTED** [review]

**Review by osherdp — APPROVED** [review]

**Review by danmanor — APPROVED** [review]

**Pull request merged**